.gitlab-ci.yml

```diff
@@ -149,6 +148,14 @@ CC7:
     # graphical libraries needed for rrdtool
     - yum install -y freetype fontconfig pixman libXrender
 
+CC8:
```

Comment:
If we know they are anyway going to fail for a while, is it worth keeping them ? It makes the process even longer

Reply:
I've done that for the integration tests but I don't see the harm in keeping this one as it's not that slow. It's arguably more useful than the fedora and ubuntu checks.

Reply:
I would actually not put any of these 3 :D But fair enough :)